Automattic/simplenote-ios
# Update checklist icon assets

### Fix
Currently, the checklist icons use .png images, this is a PR to swap them to .pdf. Checklist style has also been updated for a standard checklist icon from the Simplenote icon set. 

- Swap unchecked icon
- Swap checked icon

Before             |  After
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/36532468/76306155-4c33e880-62be-11ea-9a3d-53bd36f1e34c.png)  |  ![](https://user-images.githubusercontent.com/36532468/76312396-e1d57500-62ca-11ea-9ba5-935073273ba5.png)


Before             |  After
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/36532468/76306260-7f767780-62be-11ea-88e5-63a34a2972af.png)  |  ![](https://user-images.githubusercontent.com/36532468/76312416-edc13700-62ca-11ea-88e6-277cc3149ca5.png)


Before             |  After
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/36532468/76306457-d419f280-62be-11ea-9a12-1d92d768a35c.png)  |  ![](https://user-images.githubusercontent.com/36532468/76312432-f9146280-62ca-11ea-99c0-841e26b6f04f.png)

### Test
1. Create a checklist in a note
2. Notice that the checklist unchecked icon has changed
3. Check the item by tapping on it
4. Notice that the checklist checked icon has changed
5. Return to the notes list
6. Notice that the checklist icons have changed in the note table cell
7. Test light/dark themes

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
